docs/modules/mapstore/pages/configuring-a-generic-mapstore.adoc

```diff
@@ -257,7 +257,7 @@ mapConfig.setMapStoreConfig(mapStoreConfig);
 ====
 
 |[[id-column]]`id-column`
-|Name of the column that contains the primary key.
+|Name of the column that contains the primary key. Do not specify a column without a primary key constraint as the id-column.
```

Comment:
@Edizc does it really need to be primary key constraint? Wouldn't limiting to any unique key be enough?

Reply:
@frant-hartm you are right. I have tested it and limiting to any unique key is enough.